packages/malloy-render/src/component/table.ts

```diff
@@ -88,10 +89,11 @@ const renderFieldContent = (
       .rowLimit=${options.pinnedHeader ? 1 : Infinity}
     ></malloy-table>`;
   }
-  let value = row.cell(f).value;
+  let value: number | string = row.cell(f).value as number;
```

Comment:
I wonder if it's possible to write a guard for these values so that it's not necessary to make assumptions about their types.